lib/iris/tests/integration/test_new_axis.py

```diff
@@ -38,6 +38,15 @@ def test_lazy_data(self):
         self.assertTrue(new_cube.has_lazy_data())
         self.assertEqual(new_cube.shape, (1,) + cube.shape)
 
+    def test_masked_unit_array(self):
```

Comment:
This test should be a unit test (in the location you had it previously) not an integration test.